### PR TITLE
Implement Android 13 inverted FlatList workaround (HOT-391)

### PR DESCRIPTION
**Pull request by janisievins**
This PR implements:
- Use non-inverted FlatList on Android 13 to improve UI performance and prevent the app from freezing

See the original React Native issue here: https://github.com/facebook/react-native/issues/34583.

### CODE REVIEW

File: src/components/BubbleWrapper.tsx [hunk 1]
```diff
@@ -191,7 +192,6 @@ const BubbleWrapper = (props: PropsWithChildren<ChatbotBubbleWrapperPropsType> &
             </Animated.View>
           </Animated.View>
         </Animated.View>
-        <Animated.View style={[animatedSpacerStyles]} />
```
Comment (CharlesMangwa) [inline src/components/BubbleWrapper.tsx:194]: @janisievins why did you move this component to the top of the render tree?

File: src/components/BubbleWrapper.tsx [hunk 2]
```diff
@@ -230,7 +231,6 @@ const BubbleWrapper = (props: PropsWithChildren<ChatbotBubbleWrapperPropsType> &
           </Animated.View>
         </Animated.View>
       </Animated.View>
-      <Animated.View style={[animatedSpacerStyles]} />
```
Comment (CharlesMangwa) [inline src/components/BubbleWrapper.tsx:233]: @janisievins same question here 🤔

File: src/components/BubbleWrapper.tsx (hunk 1 above)
Comment (janisievins) [reply src/components/BubbleWrapper.tsx:194]: @CharlesMangwa actually I'm not sure why it works before my changes, because it's rendered at the bottom of the tree despite the fact that the first element in the group needs a top!!! margin of 24, see this chunk of code:
```
 const animatedSpacerStyles = useAnimatedStyle(() => {
    const height = isFirstInGroup ? 24 : 2
    return {
      borderWidth: 1,
      height: withSpring(animateBubble.value || animateBubbleIntro.value ? height : 0, DEFAULT_SPRING_CONFIG),
    }
  })
```

Another way of accomplishing correct behavior would be keeping the spacers where they are, but flipping the height ternary to ```const height = isFirstInGroup ? 2 : 24```

Comment (CharlesMangwa) [reply src/components/BubbleWrapper.tsx:194]: oh okok, gotcha! thanks for the explanation! 🙌
